outport/notifier/eventNotifier.go

```diff
@@ -35,6 +35,7 @@ type Event struct {
 	Identifier string   `json:"identifier"`
 	Topics     [][]byte `json:"topics"`
 	Data       []byte   `json:"data"`
+	TxHash     string   `json:"txHash"`
```

Comment:
`TxHash` maybe can stay near the `Identifier` field in order to keep the same types together

Reply:
:+1: